ui/src/types/index.ts

```
@@ -1,4 +1,13 @@
-import { LegalActionDTO, PlayerActionType, PlayerDTO, GameOptionsDTO} from "@bitcoinbrisbane/block52";
+import { LegalActionDTO, PlayerActionType, PlayerDTO, GameOptionsDTO, TexasHoldemStateDTO } from "@bitcoinbrisbane/block52";
+
+// Type for the return value of useGameState hook
+export interface GameStateReturn {
+    gameState: TexasHoldemStateDTO | undefined;
+    error: Error | null;
+    isLoading: boolean;
+    refresh: () => Promise<void | TexasHoldemStateDTO | undefined>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)